core/io/net_socket.h

```diff
@@ -69,7 +69,7 @@ class NetSocket : public Reference {
 	virtual bool is_open() const = 0;
 	virtual int get_available_bytes() const = 0;
 
-	virtual void set_broadcasting_enabled(bool p_enabled) = 0;
+	virtual Error set_broadcasting_enabled(bool p_enabled) = 0;
```

Comment:
Could you also sync the docs, and possibly document possible return values and their meaning?